docker/Makefile

```diff
@@ -82,13 +77,10 @@ test: prep
 		bundle exec --gemfile $$GEMFILE \
 		rspec spec
 
-push-image: prep
-	@docker push $(NAMESPACE)/r10k:$(VERSION)
-	@docker push $(NAMESPACE)/r10k:$(VERSION)-arm64
-ifeq ($(IS_LATEST),true)
-	@docker push $(NAMESPACE)/r10k:$(LATEST_VERSION)
-	@docker push $(NAMESPACE)/r10k:$(LATEST_VERSION)-arm64
-endif
+# this will use cached output from build target if it exists
```

Comment:
Does this mean that we won't rebuild the amd64 image, if we just built it?

Reply:
Correct, the buildx builder has all the previous cached layers at its disposal... and is then capable of outputting to a variety of places (docker daemon, tarball on disk, registry, etc)